config/software/elasticsearch.rb

```diff
@@ -50,6 +50,10 @@
   delete "#{project_dir}/lib/sigar/*freebsd*"
   delete "#{project_dir}/config"
   mkdir  "#{project_dir}/plugins"
+  # by default RPMs will not include empty directories in the final packag.e
+  # ES will fail to start if this dir is not present.
+  touch  "#{project_dir}/plugins/.keep"
```

Comment:
Can we name the file `.gitkeep`? That will keep the change consistent on how we handled things like this in the past across the repo:
https://github.com/chef/omnibus-software/search?utf8=%E2%9C%93&q=.gitkeep&type=